nslsii/tests/test_redis_dict.py

```diff
@@ -58,6 +58,27 @@ def test_instantiate_no_server():
         RunEngineRedisDict(host="localhost", port=9999)
 
 
+def test__parse_message_data():
```

Comment:
Accidentally double underscore

Reply:
Actually intentional double underscore, since the method under test starts with "_".